drivers.c

```diff
@@ -65,13 +65,8 @@ static char *_sub_meta_values(char *str)
 		_nt = _nt + (_at - (str + _end));
 		_var = strchr(_t, '}');
 		_end = _var - (str);
-		_start = _at - (str + _start);
 		_tstr = strdup(_t);
-		_tok = strtok(_tstr, "{:}");
```

Comment:
clang-tidy mark that as warning because the value of `_tok` is never read, it replaced by the next `strtok` call there are 2 consecutive calls as I remove the `if-else if` block

Reply:
you are right